org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -2070,3 +2075,20 @@ SsoOpenchallengesProdAdmin:
     instanceArn: !Ref instanceArn
     principalId: !Ref OpenchallengesProdAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
+
+SsoSynapseLlmDevAdmin:
+  Type: update-stacks
+  DependsOn: SsoAdministrator
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
```

Comment:
i see that the latest one in this file is `v0.7.7`, i would use that one.

Reply:
OK